### ISSUE TITLE
softmax-loss logit extraction

### ISSUE THREAD

**Issue by inkrement** [facebookresearch/fastText#819]
When using softmax as loss, is it somehow possible to get the logits (input of softmax) as inference-output instead of the softmax output?

**Comment by Celebio** [comment]
Hi @inkrement ,
It is not clear we want to expose this to end-users.
However, in the source code you can retrieve it [here](https://github.com/facebookresearch/fastText/blob/b5b7d307274ce00ef52198fbc692ed3bd11d9856/src/loss.cc#L307), right before computing the softmax.

Best regards,
Onur
